src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1231,6 +1232,79 @@ def forward(
             encoder_attentions=outputs.encoder_attentions,
         )
 
+    def generate(
+        self,
+        inputs: Optional[torch.Tensor] = None,
+        generation_config=None,
+        logits_processor=None,
+        stopping_criteria=None,
+        prefix_allowed_tokens_fn=None,
+        synced_gpus=False,
+        return_timestamps=None,
+        task=None,
+        language=None,
+        is_multilingual=None,
+        **kwargs
+    ):
+        """
+        Parameters:
+            return_timestamps (`bool`, *optional*):
+                Whether to return timestamps for each token
+            task (`bool`, *optional*):
+                Task to use for generation, either "translate" or "transcribe"
+            language (`bool`, *optional*):
+                Language token to use for generation
+            is_multilingual (`bool`, *optional*):
+                Whether or not the model is multilingual
```

Comment:
This is missing a bi of documentation ;-) If you override the docstring, it needs to be complete :-)

Reply:
I wanted to add this to the docstring rather than overwrite! Is there a natural way for this?

Reply:
You need to use the decorate we have in utils.doc for this.

Reply:
I will open a new PR for this as the `GenerationInputDocstring` has not been defined yet.